docs/guides/Guide-Intro.md

```diff
@@ -51,10 +51,11 @@ class HomeScreen extends React.Component {
   }
 }
 
-const SimpleApp = StackNavigator({
+export default SimpleApp = StackNavigator({
   Home: { screen: HomeScreen },
 });
 
+// skip this line if using Create React Native App
```

Comment:
As someone new to React Native that would remove the initial pain points of getting up and running. Thanks @matthamil